django/thunderstore/api/cyberstorm/views/user.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.http import Http404, HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.cyberstorm.serializers import CyberstormUserSerializer
+
+User = get_user_model()
+
+
+class UserAPIView(APIView):
+    permission_classes = []
+
+    @swagger_auto_schema(
+        responses={200: CyberstormUserSerializer()},
+        operation_id="api.user",
+    )
+    def get(self, request: HttpRequest, user_identifier: str) -> HttpResponse:
+        try:
+            u = User.objects.get(
+                username=user_identifier,
+            )
+        except User.DoesNotExist:
+            raise Http404(f"Coudln't find user with the identifier {user_identifier}")
+
+        serializer = self.serialize_results(u)
+
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    def serialize_results(self, user: User):
+        return CyberstormUserSerializer(
+            {
+                "name": user.username,
+                "account_created": user.date_joined,
+                "last_active": user.last_login,
```

Comment:
Let's not include this, it's leaking information that probably shouldn't be public without user control over the matter.

Reply:
Sure 👍 I just added these, because these were in the designs 🤷

Reply:
Yeah and they might very well be included later, but we probably shouldn't do that before users can control whether to expose that information or not.